tests/app/delivery/test_send_to_providers.py

```diff
@@ -63,6 +63,31 @@ def test_should_use_pinpoint_for_sms_by_default(self, restore_provider_details,
             provider = send_to_providers.provider_to_use("sms", "1234", "+16135551234")
         assert provider.name == "pinpoint"
 
+    def test_should_use_sns_for_sms_by_default_if_partially_configured(self, restore_provider_details, notify_api):
+        with set_config_values(
+            notify_api,
```

Comment:
🤷  In theory? I believe the `notify_api` object is created for each test runner.

Now, having said that, I had to make two tests serial a while ago because the variables would change during concurrent tests. And I couldn't figure out why. 😞